rust-symcrypt/Cargo.toml

```diff
@@ -24,6 +24,7 @@ default = []
 md5 = []
 sha1 = []
 pkcs1-encrypt-decrypt = []
+dynamic = ["symcrypt-sys/dynamic"]

```

Comment:
This is the feature that most developers would be interacting with. They would set the `dynamic` feature in the `rust-symcrypt` layer, and the `dynamic` flag would propagate down to the `symcrypt-sys` layer.